Makefile

```diff
@@ -5,9 +5,17 @@ build:
 	./container.sh
 
 # Run the API server locally outside of a container
-dev: load
+dev:
 	dotenv-shell cargo run
 
 # Load all infrastructure (databases, caches, etc)
 load:
 	docker-compose up -d db
+
+# Run all migrations
+migrate:
+	dotenv-shell diesel migration run
+
+# Dump the schema from the database
+schema:
```

Comment:
An enhancement here is to dump to /tmp, md5 the results, and compare to schema.rs